SAP/luigi
# Blog pre commit generate

To test: 
1. Create or modify a blogpost (luigi/blog/ folder)
2. Stage the changed blog files 
3. Run the script from luigi root with `./scripts/hooks/generate-website-blog.sh`
4. Verify the staged changes that only the staged generated blog entry has been added from `website/landingpage/public` folder


Fixes #1413
